src/macros.jl

```diff
@@ -228,13 +229,26 @@ Note that `timeout` currently only works when tests are run with multiple worker
     @testitem "Sometimes too slow" timeout=10 begin
         @test sleep(rand(1:100))
     end
+
+If a `@testitem` needs to be skipped, then you can set the `skip` keyword.
+Either pass `skip=true` to unconditionally skip the test item, or pass `skip` an
+expression that returns a `Bool` to determine if the testitem should be skipped.
+
+    @testitem "Skip on old Julia" skip=VERSION < v"1.9" begin
+        v = [1]
+        @test 0 == @allocations sum(v)
+    end
```

Comment:
Wow, i can see how precedence makes that work, but i found this whitespace very jarring. Would you accept parentheses here?
```suggestion
    @testitem "Skip on old Julia" skip=(VERSION < v"1.9") begin
        v = [1]
        @test 0 == @allocations sum(v)
    end
```